winex.js

```diff
@@ -1,7 +1,7 @@
-const winston = require("winston/lib/winston/config");
+require("winston/lib/winston/config");
```

Comment:
This line might not be necessary? I left it in just b/c I didn't want to deal with possibly breaking something.